sensu/plugins/check-vyatta-cpu.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+#
+# Checks CPU usage
+#
+# Return WARNING if CPU usage is over 80%
+# Return CRITICAL if CPU usage is over 95%
+#
+# Dean Daskalantonakis <ddaskal@us.ibm.com>
+# Carl Hannusch <channusch@us.ibm.com>
+
+import argparse
+import re
+import subprocess
+import sys
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-ip', '--ip', help='vyatta vrrp ip',
+                        required=True)
+    parser.add_argument('-p', '--password', help='snmp password',
+                        required=True)
+    args = parser.parse_args()
+
+    cmd = "snmpget -u sensu -A %s -a md5 -l authNoPriv %s \
+           1.3.6.1.4.1.2021.11.11.0" % (args.password, args.ip)
```

Comment:
We could make use of http://easysnmp.readthedocs.io/en/latest/ instead of shelling out to snmpget.